### PR TITLE
Search

### PR DESCRIPTION
**Pull request by Maorshl**

### CODE REVIEW

File: src/constants/screens.ts [hunk 1]
```diff
@@ -13,9 +13,10 @@ type MainTabsParamsList = {
 }
 
 type HomepageStackParamList = {
-    Homepage: undefined
```
Comment (Maorshl) [inline src/constants/screens.ts:16]: For this you can do: 
`Homepage?: {searchValue: string}`

File: src/features/homepage/components/TopBarSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const TopBarSearch = ({ searchValue }: TopBarSearchProps): JSX.Element => {
+
+    return (
+        <TopBar styleProps={styles.topBar}>
+            <View style={styles.topBarLeft}>
+                <GoBackButton icon={<BackIcon />} withText={false} />
+                <AppText styleProps={styles.searchValue}>{"\"" + searchValue.toUpperCase() + "\""}</AppText>
+            </View>
+            <Pressable onPress={() => { pop() }}>
```
Comment (Maorshl) [inline src/features/homepage/components/TopBarSearch.tsx:23]: onPress={pop}

File: src/features/search/components/Search.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                                    <CancelIcon />
+                                </Pressable>
+                            </Pressable>
+                        ))}
+                    </View>
+                }
+            </MainContainer >
+        </View >
+    )
+}
```
Comment (Maorshl) [inline src/features/search/components/Search.tsx:85]: Split this return to smaller functions within this file the return should look like this: 
`<View>
{renderHeader()}
{renderSection1()}
{renderSection2()}
</View>`
etc.

File: src/constants/screens.ts (hunk 1 above)
Comment (leesharon) [reply src/constants/screens.ts:16]: is this alright as well? 
Homepage: { searchValue: string } | undefined

**Review by leesharon — APPROVED** [review]

**Pull request merged**